lpcompiler.pas

```diff
@@ -1814,7 +1832,7 @@ function TLapeCompiler.ParseType(TypeForwards: TLapeTypeForwards; addToStackOwne
       else if (Tokenizer.Tok = tk_kw_Private) then
         BaseType := ltScriptMethod;
 
-      Expect([tk_kw_Function, tk_kw_Procedure], True, False);
+      Expect([tk_kw_Function, tk_kw_Procedure, tk_kw_Operator], True, False);
```

Comment:
Is an operator function a separate function type?

```
type
  //  E.g., why would you want to use
  TAdd = operator +(a, b: Int32): Int32;
  // rather than
  TAdd = function plus(a, b: Int32): Int32;
```